ext/pdo_oci/tests/pdo_oci_attr_autocommit_2.phpt

```diff
@@ -102,11 +100,17 @@ while ($r = $s->fetch()) {
 
 echo "Done\n";
 
+?>
+--CLEAN--
+<?php
+require 'ext/pdo/tests/pdo_test.inc';
+$db = PDOTest::test_factory('ext/pdo_oci/tests/common.phpt');
+PDOTest::dropTableIfExists($db, test_pdo_oci_attr_autocommit_2");
```

Comment:
```suggestion
PDOTest::dropTableIfExists($db, "pdo_ac_tab");
```

Seems like this test didn't rewrite the table name?

Reply:
Yeah, good catch, thanks!